git-subhistory.sh

```diff
@@ -1,6 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 # http://github.com/laughinghan/git-subhistory
 
+# Request bash to report that a pipeline failed if any of the commands
+# in the pipe failed (not just the last one).  This is necessary for
+# proper error handling.
+set -o pipefail
+
```

Comment:
Thanks, I wasn't aware of the git coding guidelines.

As for this issue, I think it is more compact to encapsulate this in a shell function.  Something like:

```
say_stdout () {
    "$@"
}
```
or the quiet version:

```
say_stdout () {
   "$@" >/dev/null
}
```

Then users of the code just do:

```
say_stdout git filter-branch ... || exit $?
```